tests/test_verifylib.py

```diff
@@ -960,6 +962,14 @@ def setUpClass(cls):
         layout.sign(bob)
         layout.dump(cls.layout_double_signed_path)
 
+        # dump layout with current date expiry
+        layout = copy.deepcopy(layout_template)
+        layout.signed.expires = (
+            datetime.datetime.now() +
```

Comment:
Let's verify what expiry is set.

Reply:
Here, the layout is set to expire `now()`. Right now, you're testing with `now()` a few seconds later and the expected error is indeed returned. This doesn't, however, test the change in behavior from `<` to `<=`.